src/cryptography/hazmat/primitives/constant_time.py

```diff
@@ -5,31 +5,10 @@
 from __future__ import absolute_import, division, print_function
 
 import hmac
-import warnings
 
-from cryptography import utils
-from cryptography.hazmat.bindings._constant_time import lib
 
+def bytes_eq(a, b):
+    if not isinstance(a, bytes) or not isinstance(b, bytes):
+        raise TypeError("a and b must be bytes.")
 
-if hasattr(hmac, "compare_digest"):
-    def bytes_eq(a, b):
-        if not isinstance(a, bytes) or not isinstance(b, bytes):
-            raise TypeError("a and b must be bytes.")
-
-        return hmac.compare_digest(a, b)
-
-else:
-    warnings.warn(
-        "Support for your Python version is deprecated. The next version of "
-        "cryptography will remove support. Please upgrade to a release "
-        "(2.7.7+) that supports hmac.compare_digest as soon as possible.",
-        utils.PersistentlyDeprecated2018,
-    )
-
-    def bytes_eq(a, b):
-        if not isinstance(a, bytes) or not isinstance(b, bytes):
-            raise TypeError("a and b must be bytes.")
-
-        return lib.Cryptography_constant_time_bytes_eq(
-            a, len(a), b, len(b)
-        ) == 1
+    return hmac.compare_digest(a, b)
```

Comment:
Do we need to check in with certbot on this?

Reply:
Yeah. Probably: @bmw + @ohemorange -- Is dropping Pythons that don't have `hmac.compare_digest` (2.7.x where x < mumble mumble -- Debian Jessie / Ubuntu Trusty vintage) going to cause you trouble?

We're dropping OpenSSL 1.0.1, which I think we already confirmed with you, but without the distros that use OpenSSL 1.0.1, we don't have any Pythons old enough in our CI :-)

Reply:
I don't think this will cause us any trouble.

The only distro we still support that has older than Python 2.7.7 is CentOS/RHEL 7 where it appears RHEL backported `hmac.compare_digest`.

Thanks for checking!

Reply:
Outstanding, thanks!